externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/WithSparkSQLEngine.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.kyuubi.{KyuubiFunSuite, Utils}
 trait WithSparkSQLEngine extends KyuubiFunSuite {
   protected var spark: SparkSession = _
   protected var engine: SparkSQLEngine = _
+  protected def conf: Map[String, String] = Map.empty
```

Comment:
Can we remove this default value? then we can detect improper impl at compile time

Reply:
Removed